apps/daimo-mobile/src/view/screen/link/NoteScreen.tsx

```diff
@@ -144,7 +140,7 @@ function NoteDisplayInner({
   const nonce = useMemo(() => new DaimoNonce(nonceMetadata), [ephemeralOwner]);
 
   const isV2RecipientClaim =
-    noteStatus.contractAddress === daimoEphemeralNotesV2Address &&
+    noteStatus.link.type === "notev2" &&
```

Comment:
👍 these are actually not equivalent (there were two migrations contract v1 > v2 and link v1 > v2 independently) but its fine.